### PR TITLE
WNP 64

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
WNP promoting Firefox Account features/benefits

## Issue / Bugzilla link
#6492

### CODE REVIEW

**Comment by amychurchwell** [comment]
Most recent mockup [via: @mrkwvr ]
wnp64-11262018

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ {% endblock %}
+
+ {% block site_header %}{% endblock %}
+
+ {% block content %}
+
+  <h1>{{_('Now take your Firefox to the next level: get a Firefox Account')}}</h1>
+  <h2>{{_('One log-in gives you access to all these benefits')}}</h2>
+  <p>{{_('Create a Firefox Account')}}</p>
+  <p>{{_('Have an account? Log in')}}</p>
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html]: "log in" will be a separate link. I can wrap in <a href> if affects l10n.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <h3>{{_('Put quality content in your Pocket')}}</h3>
+  <p>{{_('Discover the web’s best content, and absorb it anytime–even offline–on any device. Pocket’s listen feature will even read any article aloud to you. Get it all from the Firefox toolbar or the Pocket app.')}}</p>
+  <p>{{_('Get the Pocket App')}}</p>
+
+  <h3>{{_('Pick up where you left off with Notes')}}</h3>
+  <p>{{_('Your ideas and inspiration are secure and encrypted with Notes – and when you’re signed into your Account, they sync from your desktop to your Android devices.')}}</p>
+  <p>{{_('Get the Notes App')}}</p>
+
+  <h3>{{_('Send tabs instantly to your devices')}}</h3>
+  <p>{{_('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.')}}</p>
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html]: No CTA for send tabs. :+1:

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx64.html]: Yeah, we can do it a few possible ways... we could make the link part of the string, like:
```suggestion
  <p>{% trans login='#' %}Have an account? <a href="{{ login }}">Log in</a>{% endtrans %}</p>
```
Or make it two strings, like:
```suggestion
  <p>{{ _('Have an account?') <a href="#">{{ _('Log in') }}</a></p>
```

I might favor doing it the first way so translators have a little more freedom to structure it in a way that makes sense for their grammar.

**Comment by amychurchwell** [comment]
@peiying2 copy is finalized and l10n wrapped. Please let me know if you have any questions, thank you for your help. 🙂

**Comment by stephaniehobson** [comment]
Closing in favour of #6545.

@peiying2 the other PR is built on top of this one so I assume it's okay but please let me know if I've caused any l10n headaches!

**Pull request closed without merging**